app/views/help/new_features.md

```diff
@@ -0,0 +1,8 @@
+<!-- When updating this file, update date to flag users of new content -->
```

Comment:
It's just text and there is a little bit of ERB in there. 

The idea is, whoever updates that file just has to update the date, and the users will be notified there is new content there.